### PR TITLE
fix response type

### PR DESCRIPTION
**Pull request by alihm**
Adjustments on response type

### CODE REVIEW

File: ZelBack/src/services/syncthingService.js [hunk 1]
```diff
@@ -1723,11 +1723,20 @@ async function debugCpuprof(req, res) {
...
   let response = null;
   if (authorized === true) {
-    response = await performRequest('get', '/rest/debug/cpuprof', undefined, 60000);
+    try {
+      response = await axios.get('/rest/debug/cpuprof', {
+        responseType: 'stream', // Specify response type as stream
+        timeout: 60000,
+      });
+      res.setHeader('Content-Type', 'application/octet-stream');
+      return response.data.pipe(res);
```
Comment (TheTrunk) [inline ZelBack/src/services/syncthingService.js:1732]: was this tested?

File: ZelBack/src/services/syncthingService.js [hunk 2]
```diff
@@ -1757,11 +1775,15 @@ async function debugSupport(req, res) {
   const authorized = res ? await verificationHelper.verifyPrivilege('adminandfluxteam', req) : true;
   let response = null;
   if (authorized === true) {
-    response = await performRequest('get', '/rest/debug/support', undefined, 60000);
+    try {
+      response = await performRequest('get', '/rest/debug/support', undefined, 60000);
```
Comment (TheTrunk) [inline ZelBack/src/services/syncthingService.js:1779]: performRequest is has already try catch and practically can't throw

**Review by TheTrunk — CHANGES_REQUESTED** [review]

File: ZelBack/src/services/syncthingService.js (hunk 1 above)
Comment (alihm) [reply ZelBack/src/services/syncthingService.js:1736]: yes, it pipes the response from Syncthing.

File: ZelBack/src/services/syncthingService.js (hunk 2 above)
Comment (alihm) [reply ZelBack/src/services/syncthingService.js]: Oh, fixed it.

**Pull request merged**